### ISSUE TITLE
[Tabs View] tabs hang when "eager load tabs = false"

### ISSUE THREAD

**Issue by davetsay** [nasa/openmct#3864]
## Overview
Complex tab views can hang and get stuck loading when `eager load tabs` is set to `false`.

## Expected Behavior
Complex tab views should still load whether `eager load tabs` is set to `true` or `false`.

## Steps to Reproduce
1. Create a complex tabs view with several tabs, containing multiple types of content.
2. Set `eager load tabs` to `false` and observe load
3. Set `eager load tabs` to `true` and observe load

See user notes below for more information:
"Initially described this as an issue when Eager Load Tabs=False in a tabs view. However, I also intermittently see it when it's set to true.

Occasionally Vista will hang prior to loading plots in a tab view, note that it's hanging on loading the plot object, not the query+display of telemetry. Behavior is inconsistent and hard to pin down, but occurs most on large tab views (15+ tabs) with multiple types of content (stacked/flexible/overlay etc) spread over the tabs."

**Comment by kobe1104** [comment]
Hi @davetsay I tried to replicate this bug on hulk, but I didn't see the issue. I think we need more details on the types of tabs, content on the tabs and maybe the order the tabs were added. 

In `tabs.vue` I noticed if the tab is not added to `this.loadedTabs` then the tab view would not load when you select the tab.
It could be that the tab is somehow not added to `this.loadedTabs`. But if it's showing a loading icon (but never load) then it could be the issue of that particular domain object.
